### PR TITLE
Add script to publish to quay.io

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

**What this PR does / why we need it**:

After moving to prow, we no longer are publishing containers at quay.io when we merge to master, this job add the script to do so it will be called from prow https://github.com/kubevirt/project-infra/pull/298

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by oshoval** [comment]
LGTM

**Comment by oshoval** [comment]
here if needed, if it helps, there are the commands and the fields that will let prow access quay.io
https://github.com/kubevirt/project-infra/pull/295

**Comment by phoracek** [comment]
Also please drop the travis config file, so we don't build and push twice.

File: automation/publish.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -xe
+
+# This script publish kubernetes-nmstate-handler by default at quay.io/nmstate
+# organization to publish elsewhere export the following env vars
+# IMAGE_REGISTRY
+# IMAGE_REPO
```
Comment (phoracek) [inline automation/publish.sh]: We also need to document the GitHub token

Comment (phoracek) [reply automation/publish.sh]: Nevermind, this is only master.

**Comment by phoracek** [comment]
/lgtm
/approve
/hold

Cancel the hold when ready.

**Pull request merged**